### PR TITLE
[zh-cn] Update HTML details elements

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

get rid of half localized pages

### CODE REVIEW

File: files/zh-cn/web/html/element/details/index.md [hunk 1]
```diff
@@ -5,146 +5,93 @@ slug: Web/HTML/Element/details
...
 ## 属性
 
-此元素仅包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
+此元素包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
 
 - `open`
-  - : This Boolean attribute indicates whether or not the details — that is, the contents of the `<details>` element — are currently visible. The default, `false`, means the details are not visible.
+
+  - : 这个布尔属性表示细节内容——也就是 `<details>` 元素的内容——目前是否可见。默认情况下为 `false`，意味着细节内容是不可见的。
+
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/details/index.md:31]:
```suggestion

```

File: files/zh-cn/web/html/element/details/index.md [hunk 2]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
-**HTML `<details>`元素**可创建一个挂件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
+[HTML](/zh-CN/docs/Web/HTML) **`<details>`** 元素可创建一个组件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
 
-A disclosure widget is typically presented onscreen using a small triangle which rotates (or twists) to indicate open/closed status, with a label next to the triangle. If the first child of the `<details>` element is a `<summary>`, the contents of the `<summary>` element are used as the label for the disclosure widget.
+展现组件通常在屏幕上使用一个小三角形，旋转（或扭曲）以表示打开/关闭的状态，三角形旁边有一个标签。`<summary>` 元素的内容被用来作为展示小部件的标签。
 
 {{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-standard")}}
 
-> **备注：** The common use of a triangle which rotates or twists around to represent opening or closing the widget is why these are sometimes called "twisties."
+一个 `<details>` 小组件可以处于两种状态之一。默认的*关闭*状态只显示三角形和 `<summary>` 内的标签（如果没有`<summary>`则显示{{Glossary("user agent"，“用户代理”)}}定义的默认字符串）。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/details/index.md:14]:
```suggestion
一个 `<details>` 小组件可以处于两种状态之一。默认的*关闭*状态只显示三角形和 `<summary>` 内的标签（如果没有`<summary>`则显示{{Glossary("user agent"，"用户代理")}}定义的默认字符串）。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 3]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
-**HTML `<details>`元素**可创建一个挂件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
+[HTML](/zh-CN/docs/Web/HTML) **`<details>`** 元素可创建一个组件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
 
-A disclosure widget is typically presented onscreen using a small triangle which rotates (or twists) to indicate open/closed status, with a label next to the triangle. If the first child of the `<details>` element is a `<summary>`, the contents of the `<summary>` element are used as the label for the disclosure widget.
+展现组件通常在屏幕上使用一个小三角形，旋转（或扭曲）以表示打开/关闭的状态，三角形旁边有一个标签。`<summary>` 元素的内容被用来作为展示小部件的标签。
 
 {{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-standard")}}
 
-> **备注：** The common use of a triangle which rotates or twists around to represent opening or closing the widget is why these are sometimes called "twisties."
+一个 `<details>` 小组件可以处于两种状态之一。默认的*关闭*状态只显示三角形和 `<summary>` 内的标签（如果没有`<summary>`则显示{{Glossary("user agent"，"用户代理")}}定义的默认字符串）。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:14]:
```suggestion
一个 `<details>` 小组件可以处于两种状态之一。默认的*关闭*状态只显示三角形和 `<summary>` 内的标签（如果没有 `<summary>` 则显示{{Glossary("user agent", "用户代理")}}定义的默认字符串）。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 4]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
 
 {{HTMLSidebar}}
 
-**HTML `<details>`元素**可创建一个挂件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
+[HTML](/zh-CN/docs/Web/HTML) **`<details>`** 元素可创建一个组件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
 
-A disclosure widget is typically presented onscreen using a small triangle which rotates (or twists) to indicate open/closed status, with a label next to the triangle. If the first child of the `<details>` element is a `<summary>`, the contents of the `<summary>` element are used as the label for the disclosure widget.
+展现组件通常在屏幕上使用一个小三角形，旋转（或扭曲）以表示打开/关闭的状态，三角形旁边有一个标签。`<summary>` 元素的内容被用来作为展示小部件的标签。
 
 {{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-standard")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/details/index.md:12]:
```suggestion
{{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-shorter")}}
```

File: files/zh-cn/web/html/element/details/index.md [hunk 5]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
 ## 属性
 
-此元素仅包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
+此元素包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
 
 - `open`
-  - : This Boolean attribute indicates whether or not the details — that is, the contents of the `<details>` element — are currently visible. The default, `false`, means the details are not visible.
+
+  - : 这个布尔属性表示细节内容——也就是 `<details>` 元素的内容——目前是否可见。默认情况下为 `false`，意味着细节内容是不可见的。
+  > **备注：** 你必须完全删除这个属性来使细节内容隐藏。`open="false"` 使细节内容可见，因为这个属性是布尔值。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:31]: left an empty line here, or the style is broken.

```suggestion

    > **备注：** 你必须完全删除这个属性来使细节内容隐藏。`open="false"` 使细节内容可见，因为这个属性是布尔值。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 6]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
 
 {{HTMLSidebar}}
 
-**HTML `<details>`元素**可创建一个挂件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
+[HTML](/zh-CN/docs/Web/HTML) **`<details>`** 元素可创建一个组件，仅在被切换成展开状态时，它才会显示内含的信息。{{HTMLElement("summary")}} 元素可为该部件提供概要或者标签。
 
-A disclosure widget is typically presented onscreen using a small triangle which rotates (or twists) to indicate open/closed status, with a label next to the triangle. If the first child of the `<details>` element is a `<summary>`, the contents of the `<summary>` element are used as the label for the disclosure widget.
+展现组件通常在屏幕上使用一个小三角形，旋转（或扭曲）以表示打开/关闭的状态，三角形旁边有一个标签。`<summary>` 元素的内容被用来作为展示小部件的标签。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:10]: I may prefer "扭转".

```suggestion
展现组件通常在屏幕上使用一个小三角形，旋转（或扭转）以表示打开/关闭的状态，三角形旁边有一个标签。`<summary>` 元素的内容被用来作为展示小部件的标签。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 7]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
 
-{{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-standard")}}
+{{EmbedInteractiveExample("pages/tabbed/details.html", "tabbed-shorter")}}
 
-> **备注：** The common use of a triangle which rotates or twists around to represent opening or closing the widget is why these are sometimes called "twisties."
+一个 `<details>` 小组件可以处于两种状态之一。默认的*关闭*状态只显示三角形和 `<summary>` 内的标签（如果没有 `<summary>` 则显示{{Glossary("user agent", "用户代理")}}定义的默认字符串）。
 
-同一个 `<details>` 组件只能处于两种状态之一。The default _closed_ state displays only the triangle and the label inside `<summary>` (or a {{Glossary("user agent")}}-defined default string if no `<summary>`). This might look like the following:
+当用户点击小组件或聚焦它然后按空格键时，它就会“扭曲”展开，显示出其中的内容。通常使用一个旋转或扭曲的三角形来代表打开或关闭小组件，这就是为什么这些小组件有时被称为“扭曲的”。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:16]:
```suggestion
当用户点击小组件或聚焦它然后按空格键时，它就会“扭转”展开，显示出其中的内容。通常使用一个旋转或扭转的三角形来代表打开或关闭小组件，这就是为什么这些小组件有时被称为“旋钮”。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 8]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
 
-![Screenshot of closed <details> widget. A black left-pointing triangle sits to the right of the text “System Requirements”.](details-closed.png)
+你可以使用 CSS 来设计展示小组件，你可以通过设置/移除 [`open`](#open) 属性以编程方式打开和关闭该小组件。遗憾的是，目前还没有内置的方法来对打开和关闭之间的过渡进行动画处理。
 
-Here we see a standard disclosure widget with the label "System Requirements", in its default closed state. When the user clicks on the widget, or focuses it then presses the space bar, it "twists" open, revealing its contents:
+默认情况下，当关闭时，该小组件的高度只足以显示展开三角形和摘要。当打开时，它就会膨胀，显示其中包含的细节。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:20]:
```suggestion
默认情况下，当关闭时，该小组件的高度只足以显示展开三角形和摘要。当打开时，它就会展开以显示其中包含的细节。
```

File: files/zh-cn/web/html/element/details/index.md [hunk 9]
```diff
@@ -5,146 +5,92 @@ slug: Web/HTML/Element/details
...
 - `open`
-  - : This Boolean attribute indicates whether or not the details — that is, the contents of the `<details>` element — are currently visible. The default, `false`, means the details are not visible.
+
+  - : 这个布尔属性表示细节内容——也就是 `<details>` 元素的内容——目前是否可见。默认情况下为 `false`，意味着细节内容是不可见的。
+  > **备注：** 你必须完全删除这个属性来使细节内容隐藏。`open="false"` 使细节内容可见，因为这个属性是布尔值。
 
 ## 事件
 
-In addition to the usual events supported by HTML elements, the `<details>` element supports the {{domxref("HTMLDetailsElement/toggle_event", "toggle")}} event, which is dispatched to the `<details>` element whenever its state changes between open and closed. It is sent _after_ the state is changed, although if the state changes multiple times before the browser can dispatch the event, the events are coalesced so that only one is sent.
+除了 HTML 元素支持的常规事件外，`<details>` 元素还支持 {{domxref("HTMLDetailsElement/toggle_event", "toggle")}} 事件，当 `<details>` 元素的状态在打开和关闭之间变化时，该事件会被派发到该元素。该事件是在状态发生变化*后*发送的，不过如果状态在浏览器派发该事件之前发生了多次变化，那么这些事件就会聚集起来，从而只发送一个。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/details/index.md:35]:
```suggestion
除了 HTML 元素支持的常规事件外，`<details>` 元素还支持 {{domxref("HTMLDetailsElement/toggle_event", "toggle")}} 事件，当 `<details>` 元素的状态在打开和关闭之间变化时，该事件会被派发到该元素。该事件是在状态发生变化*后*发送的，不过如果状态在浏览器派发该事件之前发生了多次变化，那么这些事件就会被合并，从而只发送一个。
```

**Review by yin1999 — COMMENTED** [review]
I'm not really sure about the translation of "twisty"

**Review by yin1999 — APPROVED** [review]
🙏

**Pull request merged**